server/netty-server/src/test/scala/sttp/tapir/server/netty/NettyFutureTestServerInterpreter.scala

```diff
@@ -24,7 +24,7 @@ class NettyFutureTestServerInterpreter(eventLoopGroup: NioEventLoopGroup)(implic
       gracefulShutdownTimeout: Option[FiniteDuration] = None
   ): Resource[IO, (Port, KillSwitch)] = {
     val config =
-      NettyConfig.defaultNoStreaming
+      NettyConfig.defaultWithStreaming
```

Comment:
are we using the `defaultNoStreaming` anywhere?

Reply:
Not anymore, I will remove it in further development of this PR.